app/views/shop/messages/_customer_required.html.haml

```diff
@@ -6,7 +6,7 @@
       %p
       - if spree_current_user.nil?
         %p
-          = t('.require_login_html', login: ('<a auth="login" data-action="click->login-modal#call">' + t('.login') + '</a>').html_safe, signup: ('<a auth="signup" data-action="click->login-modal#call">' + t('.signup') + '</a>').html_safe)
+          = t('.require_login_html', login: ('<a auth="login" data-action="click->login-modal#call">' + t('.login') + '</a>').html_safe)
```

Comment:
this `auth="login"` is no more necessary since we open the modal to the login tab by default.
We should remove it to simplify the code.

Reply:
Done :+1: